src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift

```diff
@@ -59,12 +59,16 @@ class AntigenTestProfileInputViewController: UITableViewController, FooterViewHa
 			guard let cell = tableView.dequeueReusableCell(withIdentifier: AntigenTestProfileInputDescriptionCell.reuseIdentifier, for: indexPath) as? AntigenTestProfileInputDescriptionCell else {
 				fatalError("Wrong cell")
 			}
+			
+			cell.backgroundColor = .enaColor(for: .background)
```

Comment:
isn't it better to set the backgroundColor on cell level?

Reply:
Thanks! changed it to content view.